<launch>
	<node name="node_simulation" pkg="sailboat" type="simu_sailboat" output="screen" />
	<node name="node_draw_waypoint" pkg="sailboat" type="drawwaypoint" />
	<param name="boat_x" type="double" value="-10" />
	<param name="boat_y" type="double" value="0" />
	<param name="boat_head" type="double" value="1.25" />
	<param name="psi_wind" type="double" value="0" />


	<node name="node_heading" pkg="sailboat" type="heading_follow" output="screen" />
	<param name="thetabar" type="double" value="0" />
	
</launch>
